.bazelci/presubmit.yml

```diff
@@ -12,6 +12,16 @@ x_defaults:
     - "test/..."
     - "examples/..."
 
+  # Tests that need extra flags
+  # 1. Test that .symbols files are generated and bundled in .ipa file
+  package_symbols: &package_symbols
+    platform: macos
+    test_flags:
+    - "--apple_generate_dsym"
+    - "--define=apple.package_symbols=true"
```

Comment:
I hesitated to do that since it would unnecessarily slow down the builds of targets that don't need dSYMs and symbols generated.

Reply:
I can give it a test run.

Reply:
The build fails if we pass `--apple_generate_dsym` to everything, because of output paths collision. (There's a patch that attempts to fix this: https://github.com/bazelbuild/rules_apple/pull/715)